test/ng/directive/ngBindSpec.js

```diff
@@ -78,7 +78,7 @@ describe('ngBind*', function() {
       $rootScope.bind = new Date(2014, 10, 10, 0, 0, 0);
       element = $compile('<div ng-bind="bind"></div>')($rootScope);
       $rootScope.$digest();
-      expect(element.text()).toEqual('"2014-11-09T23:00:00.000Z"');
+      expect(element.text()).toBe('"' + $rootScope.bind.toJSON() + '"');
```

Comment:
Unrelated to this change but, is it me, or does it seem weird to use a property called `bind` in this test?